scripts/lib/node_cache.py

```diff
@@ -50,21 +55,28 @@ def setup_node_modules(npm_args=None, stdout=None, stderr=None, copy_modules=Fal
     for cmd in cmds:
         run(cmd, stdout=stdout, stderr=stderr)
 
-def do_npm_install(target_path, npm_args, success_stamp, stdout=None, stderr=None,
-                   copy_modules=False):
-    # type: (str, List[str], str, Optional[IO], Optional[IO], Optional[bool]) -> None
+def do_yarn_install(target_path, yarn_args, success_stamp, stdout=None, stderr=None,
+                    copy_modules=False, prefer_offline=False):
+    # type: (str, List[str], str, Optional[IO], Optional[IO], Optional[bool], Optional[bool]) -> None
     cmds = [
-        ["rm", "-rf", target_path],
```

Comment:
OK; I guess the benefit of removing this be that if download crashed partway through, rerunning could work?  If we feel good about `yarn` doing this flawlessly, I'd be OK with removing this; for me, the most important thing is consistent correctness.